src/channel.rs

```diff
@@ -4,10 +4,12 @@ use crate::{
     file::read_file,
     toolchain::{DistToolchainName, OfficialToolchainDescription},
 };
-use anyhow::{bail, Result};
+use anyhow::{anyhow, bail, Context, Result};
+use core::fmt;
```

Comment:
You can use `fmt` via `std` too - normally you only want to use `core` directly if working with `no_std` code.

Reply:
oops, I think i accidentally picked that from the import list. Thanks for the catch!